editor/src/components/canvas/canvas-strategies/strategies/flex-reparent-to-absolute-strategy.tsx

```diff
@@ -47,15 +47,6 @@ export function baseFlexReparentToAbsoluteStrategy(
     customStrategyState: CustomStrategyState,
   ): CanvasStrategy | null => {
     const selectedElements = getTargetPathsFromInteractionTarget(canvasState.interactionTarget)
```

Comment:
Should this file / strategy be renamed now?

Reply:
no, we decided that we still keep the two "...toAbsolute" reparents separate, and maybe unify them later, but they are doing different enough things to warrant a separate strategy. but now the metaStrategy decides which one sshould run, so this applicability check was redundant and useless